models/Performance.js

```diff
@@ -174,6 +126,6 @@ module.exports = class Performance {
     }
     const now = moment.utc(new Date().toJSON());
 
-    return moment(performance.openat).isBefore(moment(now)) && moment(now).isBefore(moment(performance.closeat));
+    return moment(new Date(performance.openat)).isBefore(moment(now)) && moment(now).isBefore(moment(new Date(performance.closeat)));
```

Comment:
http://stackoverflow.com/questions/21796446/postgres-case-sensitivity

I'm in it too deep now to go back and make everything case sensitive lol